pkg/execution/plugins/atlassian/iamrole/iam_role_manager.go

```diff
@@ -169,18 +169,18 @@ func defaultJSON(serviceName voyager.ServiceName) *IamPolicy {
 }
 
 func generateRoleInstance(spec *Spec) (*sc_v1b1.ServiceInstance, error) {
-	policyDocuments := []*IamPolicyDocument{}
+	policyDocuments := make([]*IamPolicyDocument, 0, len(spec.PolicySnippets))
```

Comment:
This shouldn't be a pointer - you need to allocate all this memory anyway. Allocating in a single block is more efficient.